gradle.properties

```
@@ -56,7 +56,8 @@ jmhPluginVersion=0.6.4
 jmhShadowPluginVersion=6.1.0
 
 junitVersion=4.13.2
-junit5Version=5.7.1
+junitPlatformVersion=1.7.2
+junit5Version=5.8.0-M1
```

```suggestion
junit5Version=5.7.1
```

Do you depend on any feature that is available only in 5.8.x version? If not, please don't change the version number. We usually submit separate PRs for version upgrades. In this case, don't think it's necessary. It's safer to stay with GA release instead of risking with M1.

---

yes, 5.8 adds support for test suits

---

Sounds good, reverting...